R/method-correlation.R

```diff
@@ -262,34 +242,154 @@ buildCorrelationComputeResult <- function(corrResult, data1, data2 = NULL, metho
 #'
 #' This function returns correlation coefficients for variables in one dataset against itself
 #' 
-#' @param data first dataset. A data.table
-#' @param method string defining the type of correlation to run. The currently supported values are 'spearman','pearson' and 'sparcc'
+#' @param data first dataset. A data.table or Collection object.
+#' @param method string defining the type of correlation to run. The currently supported values are 'spearman','pearson', 
+#' and for some methods/ data types 'sparcc'.
```

Comment:
is this still true? Looks like below 'sparcc' got removed from the argument options

Reply:
its in the generic, and i should probably add it back to the data.table method. bc the mbio computations package uses it. using rdname, its generally good practice to write the docs as generally as possible, which is what i was trying to do here.